CoronaTracker.Core/Models/Processings/ExternalCountries/Exceptions/ExternalCountryProcessingServiceException.cs

```diff
@@ -0,0 +1,16 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using Xeptions;
+
+namespace CoronaTracker.Core.Models.Processings.ExternalCountries.Exceptions
+{
+    public class ExternalCountryProcessingServiceException : Xeption
+    {
+        public ExternalCountryProcessingServiceException(Xeption innerException)
+            : base(message:"External country processing service error occurred, please contact support", innerException)
```

Comment:
This line is over 120 characters.

Reply:
Also remove `processing` from the exception message. we don't mention the architecture local anymore.

Reply:
Do the same for the rest.